Src/main.c

```diff
@@ -67,110 +68,144 @@ void SystemClock_Config(void);
 
 /* USER CODE END 0 */
 
-int main(void)
-{
+int main(void) {
+
+    /* USER CODE BEGIN 1 */
+
+    /* USER CODE END 1 */
+
+    /* MCU Configuration----------------------------------------------------------*/
+
+    /* Reset of all peripherals, Initializes the Flash interface and the Systick. */
+    HAL_Init();
+
+    /* USER CODE BEGIN Init */
+
+    /* USER CODE END Init */
+
+    /* Configure the system clock */
+    SystemClock_Config();
 
-  /* USER CODE BEGIN 1 */
+    /* USER CODE BEGIN SysInit */
 
-  /* USER CODE END 1 */
+    /* USER CODE END SysInit */
 
-  /* MCU Configuration----------------------------------------------------------*/
+    /* Initialize all configured peripherals */
+    MX_GPIO_Init();
+    MX_ADC_Init();
+    MX_TIM16_Init();
+    MX_USART1_UART_Init();
 
-  /* Reset of all peripherals, Initializes the Flash interface and the Systick. */
-  HAL_Init();
+    /* USER CODE BEGIN 2 */
+    debug("\n\nBootcamp starting up...");
+    debug("Compiled on %s at %s", __DATE__, __TIME__);
 
-  /* USER CODE BEGIN Init */
 
-  /* USER CODE END Init */
+    HAL_GPIO_WritePin(LED1_GPIO_Port, LED1_Pin, GPIO_PIN_SET);
 
-  /* Configure the system clock */
-  SystemClock_Config();
 
-  /* USER CODE BEGIN SysInit */
+    HAL_ADC_Start(&hadc);
+    HAL_TIM_PWM_Init(&htim16); //hopefully the right init this time
+    // is this init necesasry? ^^
+    HAL_TIM_PWM_Start(&htim16, TIM_CHANNEL_1);
+    /* USER CODE END 2 */
 
-  /* USER CODE END SysInit */
+    /* Infinite loop */
+    /* USER CODE BEGIN WHILE */
 
-  /* Initialize all configured peripherals */
-  MX_GPIO_Init();
-  MX_ADC_Init();
-  MX_TIM16_Init();
-  MX_USART1_UART_Init();
+    /* Note:
+     * input clock freq 48Mhz <- I have no clue what this means
+     * Pot voltage 0-3.3V
+     * pwm mod 1-2ms ot @50hz*/
 
-  /* USER CODE BEGIN 2 */
-  debug("\n\nBootcamp starting up...");
-  debug("Compiled on %s at %s", __DATE__, __TIME__);
+    // should these be unsigned ? I feel like they all could be
+    const unsigned int lowest = 1;
+    const unsigned int highest = 2;
+    const unsigned int cycle_const = 3000;
+    const float adc_max = 4095.0;
```

Comment:
Another tidbit, but usually when declaring consts you want to use caps lock

ex. LOWEST, HIGHEST, etc. It helps us tell the difference between constants and variables.